jobs/enthought_web-developer.html

```diff
@@ -0,0 +1,41 @@
+---
+title: Full Stack Web Developer 
+company: Enthought
+url: http://www.enthought.com/company/careers/
+location: Cambridge, UK
+contract: permanent
+contact:
+    name: Jill Cowan
+    email: jobs@enthought.com
+    phone: 
+created: !!timestamp '2017-02-06'
+tags:
+  - uk
+  - python
+  - flask
+  - django
+---
+
+# Overview
+
+This is an opportunity to be part of a small team at Enthought to build a growing set of web services, for both internal use and customer-facing SaaS. We are looking for experienced web developers to expand the current team in our Cambridge, UK office.
+
+Responsibilities will include moving our service toward Continuous Deployment, growing the feature set of our SaaS offering, and consolidating our web stack.
+
+Our current stack uses a mix of AWS, Ansible, Nginx, Python (Flask and Django), Docker and RabbitMQ
+
+# Desired Skills and Capabilities
+
+  * 3+ years of experience with web development
```

Comment:
Generally, we advise against job descriptions which specify years of experience. Some people learn faster than others, so it's almost never helpful and might dissuade a few fast learners!